ios/RCTTWVideoModule.m

```diff
@@ -36,30 +38,8 @@
 static NSString* statsReceived                = @"statsReceived";
 static NSString* networkQualityLevelsChanged  = @"networkQualityLevelsChanged";
 
-static const CMVideoDimensions kRCTTWVideoAppCameraSourceDimensions = (CMVideoDimensions){900, 720};
-
-static const int32_t kRCTTWVideoCameraSourceFrameRate = 15;
-
-TVIVideoFormat *RCTTWVideoModuleCameraSourceSelectVideoFormatBySize(AVCaptureDevice *device, CMVideoDimensions targetSize) {
-    TVIVideoFormat *selectedFormat = nil;
-    // Ordered from smallest to largest.
-    NSOrderedSet<TVIVideoFormat *> *formats = [TVICameraSource supportedFormatsForDevice:device];
-
-    for (TVIVideoFormat *format in formats) {
-        if (format.pixelFormat != TVIPixelFormatYUV420BiPlanarFullRange) {
-            continue;
-        }
-        selectedFormat = format;
-        // ^ Select whatever is available until we find one we like and short-circuit
-        CMVideoDimensions dimensions = format.dimensions;
-
-        if (dimensions.width >= targetSize.width && dimensions.height >= targetSize.height) {
-            break;
-        }
-    }
-    return selectedFormat;
-}
-
+CMVideoDimensions videoDimensions;
+NSInteger fps = 30;
```

Comment:
@andreychernikovich please rename to `videoFps` so its consistent with `videoDimensions`